From c7013f3287524591b61c3743f0396d89348e8f49 Mon Sep 17 00:00:00 2001
From: thomas-creel <137831176+thomas-creel@users.noreply.github.com>
Date: Thu, 22 Aug 2024 22:41:22 -0400
Subject: [PATCH] fix-deps

---
 lua-zip/lua-zip-0.2-0.rockspec | 21 ++++++++++++++++++++-
 1 file changed, 20 insertions(+), 1 deletion(-)

diff --git a/lua-zip/lua-zip-0.2-0.rockspec b/lua-zip/lua-zip-0.2-0.rockspec
index 0f45def..875b713 100644
--- a/lua-zip/lua-zip-0.2-0.rockspec
+++ b/lua-zip/lua-zip-0.2-0.rockspec
@@ -33,11 +33,30 @@ build = {
             "$(ZIP_LIBDIR)"
          },
          libraries = {
-            "zip"
+            "zip",
+            "zlib"
          },
          sources = {
             "lua_zip.c"
          }
       }
+   },
+   platforms = {
+      windows = {
+         modules = {
+            ["brimworks.zip"] = {
+               libraries = {
+                  "zip",
+                  "zlib",
+                  "advapi32",
+                  "kernel32",
+                  "user32"
+               },
+               defines = {
+                  "ZIP_STATIC"
+               }
+            }
+         }
+      }
    }
 }
-- 
2.44.0.windows.1

