From 5ca3d568229e5f6bce4e3d2a7a22c38f7c83bbcd Mon Sep 17 00:00:00 2001
From: Doug Crawford <doug.crawford@q-free.com>
Date: Mon, 17 Nov 2025 10:22:04 -0700
Subject: [PATCH] pt

---
 luaossl-20250929-0.rockspec | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/luaossl-20250929-0.rockspec b/luaossl-20250929-0.rockspec
index 79264a0..9ea37ec 100644
--- a/luaossl-20250929-0.rockspec
+++ b/luaossl-20250929-0.rockspec
@@ -29,10 +29,10 @@ external_dependencies = {
 	platforms = {
 		windows = {
 			OPENSSL = {
-				library = "libeay32"
+				library = "libcrypto"
 			};
 			CRYPTO = {
-				library = "ssleay32"
+				library = "libssl"
 			};
 		}
 	};
@@ -123,9 +123,11 @@ build = {
 			modules = {
 				["_openssl"] = {
 					libraries = {
-						"libeay32";
-						"ssleay32";
+						"libcrypto";
+						"libssl";
 						"ws2_32";
+						"Crypt32";
+						"User32";
 						"advapi32";
 						"kernel32";
 					};
-- 
2.49.0.windows.1

